picard/album.py

```diff
@@ -413,7 +427,7 @@ def _finalize_loading(self, error):
             if self.item.isSelected():
                 self.tagger.window.refresh_metadatabox()
 
-    def _finalize_loading_track(self, track_node, metadata, artists, va, absolutetracknumber, discpregap):
+    def _finalize_loading_track(self, track_node, metadata, artists, extrametadata=None):
```

Comment:
Just a minor thing, but `extra_metadata` is more readable than `extrametadata`

Reply:
Right, I renamed it